server/src/main/scala/ch.epfl.scala.index.server/DataRepository.scala

```diff
@@ -230,13 +235,16 @@ class DataRepository(github: Github)(private implicit val ec: ExecutionContext)
     for {
       updatedProject <- project(projectRef).map(_.map(p => form.update(p)))
       ret <- updatedProject
-        .flatMap(
-          project =>
-            project.id.map(
-              id =>
-                esClient
-                  .execute(update(id) in (indexName / projectsCollection) doc project)
-                  .map(_ => true)))
+        .flatMap { project =>
+            project.id.map { id =>
+              val esUpdate =
+                esClient.execute(update(id) in (indexName / projectsCollection) doc project)
+
+              val indexUpdate = SaveLiveData.saveProject(project, paths)
```

Comment:
Data is now directly saved to the `index` repository when users submit the edit form.

Note that data is still *read* from ES when projects are displayed by Scaladex. This might lead to inconsistencies in some weird cases.